### PR TITLE
ISPC 1.9.2

### PR DESCRIPTION
**Pull request by jsheard**

### CODE REVIEW

File: etc/config/ispc.amazon.properties [hunk 1]
```diff
@@ -1,6 +1,8 @@
-compilers=ispc191
-defaultCompiler=ispc191
-compilerType=ispc
```
Comment (mattgodbolt) [inline etc/config/ispc.amazon.properties]: Did you mean to remove the `compilerType` in this change? I think it's probably necessary?

Comment (jsheard) [reply etc/config/ispc.amazon.properties]: I think it's redundant, it's it inherited from ispc.defaults.properties?

Comment (mattgodbolt) [reply etc/config/ispc.amazon.properties]: Oh you're probably right! Thanks for clarifying :)

**Comment by mattgodbolt** [comment]
Sorry this took so long but I will bounce the 1.9.2 into production in the next hour or so. Thanks again.

**Pull request merged**